test/Winton.DomainModelling.DocumentDb.IntegrationTests/ValueObjectPersistenceTests.cs

```diff
@@ -192,7 +192,7 @@ public OtherTestValueObject(int value)
 
             public int Value { get; }
 
-            public bool Equals(OtherTestValueObject other) => Value == other?.Value;
+            public bool Equals(OtherTestValueObject? other) => Value == other?.Value;
```

Comment:
This was the error. I think in the latest version of the dotnet SDK they've started annotating some methods like `Equals` with `?` and because this class overrides it the signature was no longer compatible.